expressEngine/db/seedData.js

```diff
@@ -0,0 +1,26 @@
+import DBService from '../lib/service/db.js';
+import { User } from '../model';
+
+export default async function seed() {
+  const res = await DBService.count(User);
+
+  if (res <= 0) {
```

Comment:
use `if(!res) return;` instead of `if (res <= 0)`

Reply:
Done